### ISSUE TITLE
Rounded rectangle arc transparency

### ISSUE THREAD

**Issue by claviska** [claviska/SimpleImage#158]
Rounded rectangles are achieved by drawing lines and connecting them with arcs. For example:

---

Lines

![test-1](https://cloud.githubusercontent.com/assets/55639/21817598/9e519ad8-d732-11e6-8f65-efe63842d44b.png)

Arcs

![test](https://cloud.githubusercontent.com/assets/55639/21817604/a64bc790-d732-11e6-8440-78a97ebc86a4.png)

All together now! 💃🕺

![test-3](https://cloud.githubusercontent.com/assets/55639/21817608/aa23fd74-d732-11e6-82b7-c3a23306df75.png)

---

The problem occurs when a semi-transparent color is used. The arcs are drawn darker than the lines, and I haven't been able to determine why yet. Here's an example you can play with:

```php
$image = new \claviska\SimpleImage();
$image
  ->fromNew(320, 200, 'white')
  ->roundedRectangle(10, 10, 310, 190, 8, ['alpha' => .1]);
```

The result is the image below (note the darker corners). The problem only occurs in rounded rectangles that ARE NOT filled.

![test-2](https://cloud.githubusercontent.com/assets/55639/21817754/39eedc30-d733-11e6-84cc-25459df9f561.png)

---

- [ ] Fix dark corners when color is semi-transparent
- [ ] Make sure the fix works when `$thickness` is > 1

**Comment by maPer77** [comment]
I solved this problem, I still need to do some more tests to send.
![rectangle](https://user-images.githubusercontent.com/5252293/97224488-f4e07b00-17af-11eb-8acf-2dc515873161.png)

**Comment by claviska** [comment]
Fixed in #267
